pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -440,6 +441,37 @@
   ai_resources_desc_03: "Supports integration of Artificial Intelligence knowledge throughout K-12. Offer free professional development, webinars, conference presentations, and peer-to-peer learning opportunities."
   ai_resources_btn_aria_label_03: "Explore the AI for Teachers website"
 
+  ai_pl_webinar_hero_heading: "AI 101 for Teachers"
+  ai_pl_webinar_hero_desc: "This is a comprehensive online learning series for any teacher and educator interested in the groundbreaking world of artificial intelligence (AI) and its transformative potential in education. Partners Code.org, Khan Academy, ISTE, ETS, and TeachAI are offering engaging sessions with renowned experts that will demystify AI, explore responsible implementation, address bias, and showcase how AI-powered learning can revolutionize student outcomes. Join us on this journey of exploration and empowerment, and unlock the future of teaching with and about AI."
+
+  ai_pl_webinar_hero_presented: "Presented by"
+  ai_pl_webinar_intro_heading: "What is AI 101 for Teachers?"
+  ai_pl_webinar_intro_text: "This is a comprehensive online learning series for any teacher and educator interested in the groundbreaking world of artificial intelligence (AI) and its transformative potential in education. Partners Code.org, Khan Academy, ISTE, ETS, and TeachAI are offering engaging sessions with renowned experts that will demystify AI, explore responsible implementation, address bias, and showcase how AI-powered learning can revolutionize student outcomes. Join us on this journey of exploration and empowerment, and unlock the future of teaching with and about AI."
+  ai_pl_webinar_partnership_between: "A partnership between"
+
+  ai_pl_webinar_join_email_title: "Sign up for future AI updates"
+  ai_pl_webinar_pro_join_email_desc: "Be the first to hear about new AI opportunities from Code.org, Khan Academy, ISTE, and ETS."
+
+  ai_pl_webinar_explore_title: "Explore additional AI resources"
+  ai_pl_webinar_explore_desc: "We have a variety of additional resources for  you from Code.org, ETS, ISTEI, Khan Academy and TeachAI to support teachers in this new and growing field. Check them out below."
+
+  ai_pl_webinar_explore_tag_1: "EDUCATOR TOOL"
```

Comment:
These tag strings (line 458, 462, etc) are similar to these: https://github.com/code-dot-org/code-dot-org/blob/e6a7540e7dbbe6029b00506c08d3ded184705ad0/pegasus/sites.v3/hourofcode.com/i18n/en.yml#L253

Can you move these up to this section linked above and change the case the title case? ("Educator Tool") The uppercase styling can be applied using the `.overline` class.